src/app/modules/visualizacion-informacion/components/exportar-huds/exportar-huds.component.ts

```diff
@@ -0,0 +1,137 @@
+import { Plex } from '@andes/plex';
+import { Component, OnInit } from '@angular/core';
+import { IPaciente } from 'src/app/core/mpi/interfaces/IPaciente';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { DocumentosService } from '../../../../services/documentos.service';
+import { TipoPrestacionService } from '../../../../services/tipoPrestacion.service';
+import { Auth } from '@andes/auth';
+import { Router } from '@angular/router';
+import { ModalMotivoAccesoHudsService } from 'src/app/modules/rup/components/huds/modal-motivo-acceso-huds.service';
+
+
+@Component({
+    selector: 'app-exportar-huds',
+    templateUrl: './exportar-huds.component.html'
+})
+
+export class ExportarHudsComponent implements OnInit {
+    resultadoBusqueda = null;
+    pacienteSelected = null;
+    form: FormGroup = new FormGroup({
```

Comment:
Ojo con los warnings y errores de consola.

```
      It looks like you're using the disabled attribute with a reactive form directive. If you set disabled to true
      when you set up this control in your component class, the disabled attribute will actually be set in the DOM for
      you. We recommend using this approach to avoid 'changed after checked' errors.
       
      Example: 
      form = new FormGroup({
        first: new FormControl({value: 'Nancy', disabled: true}, Validators.required),
        last: new FormControl('Drew', Validators.required)
      });
```